### PR TITLE
Administrator

### PR DESCRIPTION
**Pull request by jKwibe**
User story 38 - 41
- Disables the merchant 
- Disables all the items of that merchant disabled.

- Enables the merchant 
- enables all the items of that merchant enabled.

### CODE REVIEW

File: db/migrate/20200728192201_add_enabled_to_merchant.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddEnabledToMerchant < ActiveRecord::Migration[5.1]
+  def change
+    add_column :merchants, :enabled, :boolean, default: true
```
Comment (sciencefixion) [inline db/migrate/20200728192201_add_enabled_to_merchant.rb:3]: Good implementation of enabled boolean!

**Comment by jKwibe** [comment]
I've added more to the PR.

File: app/controllers/admin/merchants_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class Admin::MerchantsController < Admin::BaseAdminController
```
Comment (RyN21) [inline app/controllers/admin/merchants_controller.rb:1]: Cool! Admin merchant controller

File: app/controllers/admin/merchants_controller.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    def index
+        @merchants = Merchant.all
+    end
+
+    def update
```
Comment (RyN21) [inline app/controllers/admin/merchants_controller.rb:6]: Looks good

**Pull request merged**